### ISSUE TITLE
Information panel information icon missing in some cases

### ISSUE THREAD

**Issue by paladique** [github/VisualStudio#692]
When working on #665, in some of the [screenshots](https://github.com/github/VisualStudio/pull/665#issuecomment-261030796), the information icon in the information panel is missing. 

From that example, the screenshots where the icon is missing are when I successfully created a pull request without any errors. When accidentally creating an already existing pull request for a branch, going to .com to delete it, and making the pull request again within the extension, the information icon is visible. It is also visible after I make more pull requests. 

Closing the info panel and creating a successful pull request right after will also not display the icon. Also, if I make two (or more) pull requests in a row successfully, the icon will still not show up. 

To sum this up, the icon is missing when:
- You close the info panel (if it's currently open/visible)
- The first message is information (successful creation, not an error)

It's not missing when:
- The first message was a warning or error

**Comment by paladique** [comment]
Doing a little digging into this reveals that `UpdateIcon` in `InfoPanel.xaml.cs` is not getting called when the message type is `MessageType.Information`

**Comment by paladique** [comment]
One thing I did notice was that on line 37
```
get { return (MessageType)GetValue(MessageProperty); }
```
should probably be MessageTypeProperty instead, but that didn't seem to change anything

**Comment by tocsoft** [comment]
It'll be because the default value is `MessageType.Information` thus its value is never changing therefore the propertyChanged callback is never being called thus the `Icon` and `IconColor` are never set. Just set them in the constructor to the same values you would get for `MessageType.Information` and I think it'll work.
